app/components/Nav/Mobile/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import {Link} from '@remix-run/react'
+import AISafetyIcon from '~/components/icons-generated/Aisafety'
+import {ListLarge} from '~/components/icons-generated'
+import {XLarge} from '~/components/icons-generated'
+import {CarrotLarge} from '~/components/icons-generated'
+import OpenBookIcon from '~/components/icons-generated/OpenBook'
+import BotIcon from '~/components/icons-generated/Bot'
+import {NavProps} from '~/components/Nav'
+import Button from '~/components/Button'
+import '../nav.css'
+import './navMobile.css'
+import {sortFuncs} from '~/routes/tags.$'
+import {questionUrl, tagsUrl, tagUrl} from '~/routesMapper'
+import {TOCItem} from '~/routes/questions.toc'
+export const MobileNav = ({toc, categories}: NavProps) => {
+  const [showMenu, setShowMenu] = React.useState(false)
+  const [showArticles, setShowArticles] = React.useState(false)
+  const toggleMenu = () => {
+    setShowMenu(!showMenu)
+    setShowArticles(false)
+  }
+  console.log('toc', toc)
+  return (
+    <header className="top-header">
+      {!showArticles && (
+        <>
+          <nav className="top-nav">
```

Comment:
how much of this could be handled with css in the normal `Nav` component?

Reply:
I'm not sure I follow, can you please explain what you mean?

Reply:
I only just noticed that I didn't send this comment...

Reply:
preferably. But this certainly still needs to be connected to the actual page